Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TetrahedralCorotationalFEMForceField.h

```diff
@@ -190,12 +192,22 @@ class TetrahedralCorotationalFEMForceField : public BaseLinearElasticityFEMForce
     Data<sofa::type::RGBAColor> d_drawColor3; ///<  draw color for faces 3
     Data<sofa::type::RGBAColor> d_drawColor4; ///<  draw color for faces 4
     Data<std::map < std::string, sofa::type::vector<double> > > _volumeGraph;
+    
+    Data<bool> d_computeVonMisesStress;
+    Data<type::vector<Real> > d_vonMisesPerElement; ///< von Mises Stress per element
+    Data<type::vector<Real> > d_vonMisesPerNode; ///< von Mises Stress per node
+    
+    sofa::helper::ColorMap* m_VonMisesColorMap;
```

Comment:
```suggestion
    Data<type::vector<Real> > d_vonMisesPerNode; ///< von Mises Stress per node
    
    sofa::helper::ColorMap* m_vonMisesColorMap;
```
I guess this is a copy-paste from the other file 🫢but it is nice to have consistency